run_dir/static/js/project_samples.js

```diff
@@ -1232,3 +1233,26 @@ function old_project_warning(warndate_raw){
     $('#old_project_warning').attr('title', 'This project was created before '+warndate_raw+'.<br>Genomics Status may be inaccruate.');
   }
 }
+function load_charon_summary(){
+  $.getJSON("/api/v1/charon_summary/"+ project, function(data) {
+      if (data['tot']==0){
+          $('#tab_charon_data').html("This project has no data in Charon.");
+      }else{
+          table="<table class='table table-bordered table-striped'>";
+          table+="<tr><td>Total amount of samples</td><td>"+data['tot']+"</td></tr>";
+          table+="<tr><td>Sequenced samples</td><td>"+data['seq']+"</td></tr>";
+          table+="<tr><td>Analyzed samples</td><td>"+data['ana']+"</td></tr>";
+          table+="<tr><td>Successful Analysis</td><td>"+data['passed']+"</td></tr>";
+          table+="<tr><td>Failed Analysis</td><td>"+data['failed']+"</td></tr>";
+          table+="<tr><td>Running Analysis</td><td>"+data['runn']+"</td></tr>";
+          table+="<tr><td>Number of Human Genomes covered</td><td>"+data['hge']+"</td></tr>";
+          table+="<table>";
+          $('#tab_charon_data').html(table);
```

Comment:
Might be nicer to use a [horizontal description list](http://getbootstrap.com/css/#horizontal-description) as in the top of the page.

Also, numbers should be in [badges](http://getbootstrap.com/components/#badges) really, for visual consistency.

```javascript
$('#tab_charon_data').html(
    '<dl class="dl-horizontal">
        <dt>Total amount of samples</dt>  <dd><span class="badge">'+data['tot']+'</span></dd>
        <dt>Sequenced samples</dt> <dd><span class="badge">'+data['seq']+'</span></dd>
        <dt>Analyzed samples</dt> <dd><span class="badge">'+data['ana']+'</span></dd>
        <dt>Successful Analysis</dt> <dd><span class="badge label-success">'+data['passed']+'</span></dd>
        <dt>Failed Analysis</dt> <dd><span class="badge label-danger">'+data['failed']+'</span></dd>
        <dt>Running Analysis</dt> <dd><span class="badge">'+data['runn']+'</span></dd>
        <dt>Number of Human Genomes covered</dt> <dd><span class="badge">'+data['hge']+'</span></dd>
    </dl>');
```